login/login.go

```diff
@@ -0,0 +1,384 @@
+package login
+
+import (
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"gopkg.in/ini.v1"
+)
+
+// Help text and command line flags.
+
+// Usage text that will be displayed as command line help text when using the
+// `help login` command
+var Usage = `
+
+USAGE: %s login <login-target>
+
+login:
+    logs in to the SDA using the provided login target.
+`
+
+// ArgHelp is the suffix text that will be displayed after the argument list in
+// the module help
+var ArgHelp = `
+    [login-target]
+        The login target can be one of the following: bp.nbis.se`
```

Comment:
Here is the inconsistency regarding the URL

Reply:
fixed